File: src/main/java/pe/com/test_qsystem/distribution_center/impl/IcaDistributionCenterImpl.java
package pe.com.test_qsystem.distribution_center.impl;

import pe.com.test_qsystem.distribution_center.DistributionCenter;

public class IcaDistributionCenterImpl implements DistributionCenter {

    @Override
    public boolean support(String center) {
        return center.equalsIgnoreCase("ICA");
    }

    @Override
    public void operate(String center) {
        System.out.println("Operate in Ica distribution center.");
    }
}

File: src/main/kotlin/pe/com/qsystem/test_qsystem/distribution_center/impl/IcaDistributionCenterImpl.kt
package pe.com.qsystem.test_qsystem.distribution_center.impl

import org.slf4j.LoggerFactory
import pe.com.qsystem.test_qsystem.distribution_center.DistributionCenter

class IcaDistributionCenterImpl : DistributionCenter {
        
    private val log = LoggerFactory.getLogger(javaClass)

    override fun support(center: String) = center.contentEquals("ICA", true)

    override fun operate(center: String) {
        log.info("Operate in Ica distribution center.")
    }
}